.kitchen.yml

```diff
@@ -12,10 +12,8 @@ client_rb:
 verifier: inspec
 
 platforms:
-- name: ubuntu-16.04
-- name: windows-2012r2
-  driver_config:
-    box: mwrock/Windows2012R2
+- name: ubuntu-18.04
+- name: centos-7
```

Comment:
Why are we removing the windows tests?  Also testing both centos and ubuntu seems like overkill for line.

Reply:
If we're not running them in an automated fashion they are of little value.